### ISSUE TITLE
device stop doesn't handle restarting (crashing) containers

### ISSUE THREAD

**Issue by adamkdean** [edge/cli#74]
![image](https://user-images.githubusercontent.com/1639527/145689927-6d715079-2a6a-4780-b83e-99581362206d.png)

**Comment by annybs** [comment]
TL;DR I think this is misreporting by CLI due to limits of the old container matching behaviour. As of https://github.com/edge/cli/commit/9cf7413e9230c5900231734bd320af1ea54f6293 this problem no longer exists.

--

This service would have been started with `SPECIFIC_TAG=0.1.0-10 edgetest device start` meaning that subsequent interactions with the container would also require `SPECIFIC_TAG` to be set, otherwise there would be no match for the container based on image name. Ergo, in the screenshot the `edgetest device stop` looked for a container running `:latest`, found none, and reported as such.

Now that we are using container name for matching basis, the container should always be matched if it exists, even if it's stuck in a restart loop. This will be particularly useful for `update` et al. where the image name is likely to vary between versions.
